frontend/src/dep.ts

```diff
@@ -46,6 +47,7 @@ export function initUIFactory(
   const versionService = new VersionService(envService);
   const store = initStore();
   const searchService = new SearchService();
+  const updatesService = new UpdatesService();
```

Comment:
I am wondering, instead of injecting the UpdatesService into UIFactory, can we use it as a static class with static methods? I don't see any use case where we will be needing to store user specific states in this service class. What are your thoughts? @kapillamba4 @byliuyang

Reply:
@rohithbalaji123  I think it will depend on UserService or AuthService or ErrorService. User need to known in order to fetch its last seen time from backend.

Reply:
Ohkay, yeah. Makes sense!

Reply:
@rohithbalaji123 We prefer instance than static so that we can swap implementation during unit testing. We'll soon start adding unit tests.